### PR TITLE
[WIP] Transfer install shellscripts to nodescripts

### PR DESCRIPTION
**Pull request by ckanitz**
To gain OS cross-compatibility all install scripts should be rewritten/readded as node.js scripts:

- [x] setup-local-env.js
- [x] ~> install-npm.js~
- [x] > install-docker.js
- [x] > install-wpcs.js
- [ ] > install-wordpress.js
- [ ] i18n-create-pot.js (add `cp` to move the pot from `/trunk/languages/` to `/build/languages/`)
- [ ] i18n-create-json.js (add `cp`/`mv`/`rename` to move the json from `/trunk/languages/` to `/build/languages/`)
- [ ] define-env.sh

### CODE REVIEW

**Comment by ckanitz** [comment]
remove npm-install.js
the node scripts require some packages to be installed in the firstplace so the chain of executing this command would be more like `npm install && node ./bin/setup-locel-env.js`.

File: bin/install-docker.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+				} )
+				.catch( ( err ) => { this.onDockerError( err ) } );
+		} );
+	}
+
+	/**
+	 * Stop existing containers.
+	 * This stops all running containers, including those on port 80.
+	 */
+	async composeStop() {
```
Comment (Luehrsen) [inline bin/install-docker.js]: In our shell script we had a section where we stopped all containers on port 80, not just the current defined in compose. Does this do that?

**Comment by Luehrsen** [comment]
> remove npm-install.js
> the node scripts require some packages to be installed in the firstplace so the chain of executing this command would be more like `npm install && node ./bin/setup-locel-env.js`.

Maybe reference `npm install && node...` as `npm start`. That seems to be the appropriate default command.

File: bin/install-docker.js (hunk 1 above)
Comment (ckanitz) [reply bin/install-docker.js]: This should shut down every running container. I tested it by starting several containers and every single one gut shut down.

The main problem to implement the extra check for port 80 is `curl` W hoch is no default command on windows. Might need to research a package that could wrap curl or similiar if the check is neccessary.

**Comment by Luehrsen** [comment]
https://github.com/WordPress/gutenberg/blob/master/packages/scripts/scripts/env/docker-run.js

**Pull request closed without merging**